### PR TITLE
Initial changes to include TFB data & Kruize setup

### PR DESCRIPTION
**Pull request by chandrams**
Signed-off-by: Chandrakala Subramanyam <csubrama@redhat.com>

### CODE REVIEW

File: remote_monitoring_demo/update_results.json [hunk 1]
```diff
@@ -34,16 +32,16 @@
          "results": {
            "general_info": {
              "max": 0.84,
-             "mean": 0.12,
+             "avg": 0.12,
```
Comment (dinogun) [inline remote_monitoring_demo/update_results.json:35]: cpuUsage needs to have `min`, `max`, `avg` and `sum`.
cpuThrottle needs to have `max`, `avg` and `sum`.

File: remote_monitoring_demo/update_results.json [hunk 2]
```diff
@@ -70,7 +68,8 @@
          "results": {
            "general_info": {
              "max": 198.50,
-             "mean": 40.1,
+             "avg": 40.1,
```
Comment (dinogun) [inline remote_monitoring_demo/update_results.json:71]: Both `memoryUsage` and `memoryRSS` needs to have `min`, `max`, `avg` and `sum`

File: remote_monitoring_demo/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Pre-req
+It expects minikube to be installed with atleast 8 CPUs and 16384MB Memory.
+**WARNING:** The script deletes any existing minikube cluster.
+
+## How do I run it?
+
+```
+# Run the Kruize monitoring demo using the below command:
+$ ./monitoring_demo.sh
```
Comment (dinogun) [inline remote_monitoring_demo/README.md:23]: Now that this has been renamed, change it to `remote_monitoring_demo.sh` in all places

Comment (chandrams) [reply remote_monitoring_demo/README.md]: Updated, thanks.

File: remote_monitoring_demo/update_results.json (hunk 2 above)
Comment (chandrams) [reply remote_monitoring_demo/update_results.json]: Done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**